docs/HowTo/Get-Started/Build-from-source.md

```diff
@@ -0,0 +1,10 @@
+---
+description: Install or Hyperledger Besu from source
+---
+
+# Build from Source
+
+View the [Hyperledger documentation] for instructions to install Hyperledger Besu from source.
```

Comment:
Maybe specify that it's recommended only if you need to use the latest dev version from master or a specific commit, but that for regular use the pre-compiled binary and Docker images are simpler and faster to use and more stable.

Reply:
Done